Docs/source/building/spack.rst

```diff
@@ -1,40 +1,49 @@
 Building WarpX with Spack
 ===============================
 
-WarpX can be installed using Spack. From the Spack web page: "Spack is a package management tool designed to support multiple
-versions and configurations of software on a wide variety of platforms and environments."
+WarpX can be installed using Spack.
+From the `Spack web page <https://spack.io>`_: "Spack is a package management tool designed to support multiple versions and configurations of software on a wide variety of platforms and environments."
 
-Spack is available from `github <https://github.com/spack/spack>`__. Spack only needs to be cloned and can be used right away - there are no installation
-steps. The spack command, "spack/bin/spack", can be used directly or "spack/bin" can be added to your execute path.
+.. note::
+
+   Quick-start hint for macOS users:
+   Before getting started with Spack, please check what you manually installed in ``/usr/local``.
+   If you find entries in ``bin/``, ``lib/`` et al. that look like you manually installed MPI, HDF5 or other software at some point, then remove those files first.
+
+   If you find software such as MPI in the same directories that are shown as symbolic links then it is likely you `brew installed <https://brew.sh>`_ software before.
+   Run `brew unlink ... <https://docs.brew.sh/Tips-N%27-Tricks#quickly-remove-something-from-usrlocal>`_ on such packages first to avoid software incompatibilities.
+
+Spack is available from `github <https://github.com/spack/spack>`_.
+Spack only needs to be cloned and can be used right away - there are no installation steps.
```

Comment:
Maybe we can move to here the link to the spack tutorial (it is super well done!)

Reply:
Good idea, you mean the one here? https://spack-tutorial.readthedocs.io

Reply:
Yes, exactly :)

Reply:
Thanks for all the changes and for explaining everything Axel

Reply:
I don't see the approve buttons. But I think as soon as you mark this conversation as resolved, they will come back and I will be able to approve this PR.